graf2d/gpadv7/v7/inc/ROOT/RCanvas.hxx

```diff
@@ -86,9 +86,12 @@ public:
    /// Display the canvas.
    void Show(const std::string &where = "");
 
-   /// Close all canvas displays
+   /// Hide all canvas displays
    void Hide();
 
+   /// Remove canvas from global canvas lists, will be destroyed when shared_ptr will be removed
+   void Remove();
```

Comment:
Maybe `Unregister()`? Or is `Remove()` a common or previous name for this operation?